packages/transcription/transcribe/README.md

```diff
@@ -0,0 +1,19 @@
+## TU Capstone- Transcription
```

Comment:
Very cool!

Reply:
thanks! it's not 100% working, but as we proceed with building other little features we're going to open this back up and start using it more